app/assets/stylesheets/app.scss

```diff
@@ -19,6 +19,18 @@ html {
 
 body {
   margin: 0;
+  position: relative;
+  padding-bottom: 209px;
```

Comment:
why is this 209px?

Reply:
Its the height of the footer. i used the same so that the content don't overlap with the footer